rasa/engine/caching.py

```diff
@@ -304,10 +304,14 @@ def _is_disabled(self) -> bool:
     def _cache_output_to_disk(
         self, output: Cacheable, model_storage: ModelStorage
     ) -> Tuple[Optional[Text], Optional[Text]]:
+        tempdir_name = tempfile.mkdtemp()
+        if isinstance(tempdir_name, bytes):
+            tempdir_name = tempdir_name.decode("UTF-8")
```

Comment:
🤔 weird, I can't find info about this behaviour change in the release notes

Reply:
I know 😬 The `mkdtemp()` function returns type AnyStr, which includes bytes, not sure if that's been changed in 3.10.